src/components/sbb-map-container/sbb-map-container.scss

```diff
@@ -0,0 +1,71 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-map-container-map-height-tiny: #{sbb.px-to-rem-build(295)};
+  --sbb-map-container-map-height-small: #{sbb.px-to-rem-build(320)};
+  --sbb-map-container-sidebar-width: #{sbb.px-to-rem-build(400)};
+  --sbb-map-container-sidebar-background-color: var(--sbb-color-white-default);
+}
+
+.sbb-map-container {
+  display: grid;
+  grid-template-rows: auto auto;
+  height: 100%;
+
+  @include sbb.mq($from: medium) {
+    grid-template-columns: max(var(--sbb-map-container-sidebar-width)) calc(
+        100% - var(--sbb-map-container-sidebar-width)
+      );
+    width: 100vw;
+    height: calc(100vh - var(--sbb-header-height));
```

Comment:
Following suggestion:
replace width and height with the following, add to the documentation that spacing can be overriden with --sbb-map-container-margin-start if needed. So it still defaults to the header height for most of the scenarios, but can be configured without introducing a property for that. And it can also be used for whatever size needed.
```
    position: absolute;
    inset: 0px;
    padding-block-start: var(--sbb-map-container-margin-start, var(--sbb-header-height));
```